source/docs/contribute/documentation.md

```diff
@@ -54,13 +54,25 @@ All images of the HACS documentation are located under:
 source/assets/images/
 ```
 
+## Creating new pages
 
-## Contents linking
+1. Create a new `.md` file under `source/docs/` in a subdirectory that fits the purpose of the file.
+2. Add a reference to that file in `mkdocs.yml`.
 
-Instructions on how to create references in Markdown.
+At the top of all pages, add a header:
 
+```yaml
+---
+title: A title for the page, this will also be used in the sidebar. For example, Feature.
+description: "A summary of the contents of this page"
+---
+```
 
-### Referencing another page
+## Linking in Markdown

```

Comment:
```suggestion
## Linking in the documentation

```
"Maekdown" is an implementation detail here, the goal is adding links in the documentation.